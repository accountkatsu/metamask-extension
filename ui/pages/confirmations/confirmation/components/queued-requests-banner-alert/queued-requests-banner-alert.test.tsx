import * as React from 'react';
import configureStore from 'redux-mock-store';
import mockState from '../../../../../../test/data/mock-state.json';
import { renderWithProvider } from '../../../../../../test/lib/render-helpers';
import { QueuedRequestsBannerAlert } from './queued-requests-banner-alert';

describe('<QueuedRequestsBannerAlert />', () => {
  beforeEach(() => {
    jest.resetAllMocks();
  });

  it('should render component', () => {
    const store = configureStore()(mockState);

    const { container } = renderWithProvider(
      <QueuedRequestsBannerAlert />,
      store,
    );

    expect(container).toMatchSnapshot();
  });
});
