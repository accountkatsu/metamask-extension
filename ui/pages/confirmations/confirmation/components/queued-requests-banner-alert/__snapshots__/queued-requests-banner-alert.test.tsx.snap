// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<QueuedRequestsBannerAlert /> should render component 1`] = `<div />`;
